lighthouse-core/lib/dependency-graph/network-node.js

```diff
@@ -57,6 +58,21 @@ class NetworkNode extends BaseNode {
     return !!this._record.fromDiskCache;
   }
 
+  /**
+   * @return {boolean}
+   */
+  get isNonNetworkProtocol() {
+    return URL.NON_NETWORK_PROTOCOLS.includes(this._record.protocol);
```

Comment:
Is it useful to reuse this here vs making a new list here? There are some non network protocols we wouldn't include here (presumably eg intent wouldn't end up as a node, but this list might expand in the future)

Reply:
I'm not sure when this would actually need to diverge from our canonical set, so I'd prefer to reuse. It's true it will include things we don't actually need to test but I doubt the reverse will ever be true